### DISCUSSION TITLE
Add built in work flows for ready to publish notifications.

### DISCUSSION THREAD

**Discussion by Eaglef90** [umbraco/Umbraco-CMS#16195] [Features and ideas]
I did some digging in the docs and even asked in the discord but it seems that if I want to do something as simple and basic as send an email to the appropriate editors when content has been "Send to publish" there is no built-in way to do this. The solutions provided to me where to custom-code a notification or buy a commercial addon. It would seem to me that this would be a basic and normal functionality of a CMS to give some way of notifying certain people/roles when they have work to do, i.e. review and publish content from a writter.

**Comment by bergmania** [comment]
I will any day recommend Umbraco.Workflow. I assume this is what our reference as commercial add-on, but there is a quite big feature list on the free version too https://umbraco.com/products/add-ons/workflow/

**Reply by Eaglef90** [reply]
I was never abke to get work flow to really work. While I got notofications for some reason it when new nodes where created it would make two of them and no matter what I set a users permission to it always foreced everyone to "send to publish". So I just removed it. I founf that the " notoficstion" option i. The co tent node right click menu and while not what i want it does at lease work a bit better in telling me I need to go look at things.

**Reply by nathanwoulfe** [reply]
Hey @Eaglef90 this is absolutely a core use-case for Umbraco Workflow, so I'm interested in why it wasn't a good fit. If you could let me know the Umbraco + Workflow versions you were using, as well as any configuration that might be appropriate, I'll see if I can replicate the duplicate nodes issue.

**Reply by Eaglef90** [reply]
I dont know the exact version numbers but it would have been the highest v13 available in June
